docs/commands/push-status.md

```diff
@@ -0,0 +1,64 @@
+# `push-status`
+
+This command is used with Reunite products only.
```

Comment:
Should this be an admonition? Make it stand out more?

Reply:
I try to use fewer markdoc extras in the CLI repo since anyone reading the docs on GitHub won't see that content rendered well. And I'm not sure how many people will try to use this command and find it confusing .... so I erred on the side of leaving it as words for now. We might well need to revisit later!